tools/tertiary-analysis/scanpy/scanpy-find-markers.xml

```diff
@@ -64,24 +71,26 @@ PYTHONIOENCODING=utf-8 scanpy-find-markers
 
   <outputs>
     <expand macro="output_data_obj" description="Marker genes"/>
-    <data name="output_csv" format="csv" from_work_dir="output.csv" label="${tool.name} on ${on_string}: Marker table">
+    <data name="output_tsv" format="tabular" from_work_dir="diffexp.tsv" label="${tool.name} on ${on_string}: Marker table">
```

Comment:
has this output always been tsv in practice or we are actually changing the format from commas to tabs in the output? If that is the case, it should probably be mentioned in the tool history at the end.

Reply:
It has been for some time, Ni changed it sneakily a while back.